c2corg_ui/views/document.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, request):
             'debug': 'debug' in self.request.params,
             'api_url': self.settings['api_url'],
             'ign_api_key': self.settings['ign_api_key'],
-            'bing_api_key': self.settings['bing_api_key']
+            'bing_api_key': self.settings['bing_api_key'],
+            'image_backend_url': self.settings['image_backend_url'],
+            'image_url': self.settings['image_url']
```

Comment:
Perhaps you have to add those 2 mako variables also to https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/__init__.py#L95-L100 for the 404 page. You don't really need them in the 404 page but ``templates/404.html`` also inherits from ``base.html`` https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/404.html#L1

Try to add whatever string to your instance URL and check that the 404 page is not broken.
Normally ``make check`` should report the problem as well I think.